mandatory/2-function-creation.js

```diff
@@ -25,7 +33,15 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  let arr = arrayOfStrings;
+  for (let i = 0; i < arrayOfStrings.length; i++) {
+    arr[i] = arr[i].trim();
+    arr[i] = arr[i].replace("/", "");
+    arr[i] = arr[i].toLowerCase();
+  }
+  return arr;
```

Comment:
Yes, this is correct. You also have the option of chaining the 3 functions into 1 line, but the way you have done it is more readable, so it's a matter of personal preference.

Reply:
Thanks for reviewing this weeks work Karen, really appreciate the feedback. I have made the necessary changes as you mentioned and will push them through as soon as I can.